### PR TITLE
Add Indonesian (ID) translation

### PR DESCRIPTION
**Pull request by fitraditya**
Add Indonesian (ID) translation. First chapter translated, more chapter later.

### CODE REVIEW

**Comment by hiroyuki-sato** [comment]
Hello, @fitraditya. Thank you for your contribution. I read your PR(`_index.md` and `01-what-why-and-how`) using Google Translate. It seems good.

@mogren Please take a look when you get a chance.

**Review by hiroyuki-sato — APPROVED** [review]

File: config.toml [hunk 1]
```diff
@@ -41,6 +41,11 @@ DefaultContentLanguage = "en"
     languageName = "Français"
     contentDir = "content.fr"
     weight = 6
+  [languages.fr]
```
Comment (hiroyuki-sato) [inline config.toml:44]: `languages.id` ?

**Review by hiroyuki-sato — CHANGES_REQUESTED** [review]

**Review by mogren — APPROVED** [review]
Nice @fitraditya, thanks for working on this! 🎉

**Review by hiroyuki-sato — APPROVED** [review]
@fitraditya Thanks!
@mogren Thank you for your review.

**Pull request merged**